modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -191,12 +198,14 @@ export default {
       readOnly: false,
       restoreOnly: false,
       saveMenu: null,
-      generation: 0
+      generation: 0,
+      isLocalizing: false,
+      currentId: this.docId
```

Comment:
This new currentId, allows us to switch it when switching the locale (passed to `AposSchema` for ex), we still keep `docId` as a reference to the original doc id.